lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -122,6 +141,36 @@ def test_score_set_bad_user(self, handler, kwargs):
         handler(None, **kwargs)
         self.signal_mock.assert_not_called()
 
+    def test_raw_score_changed_signal_handler(self):
+        problem_raw_score_changed_handler(None, **PROBLEM_RAW_SCORE_CHANGED_KWARGS)
+        expected_set_kwargs = {
+            'sender': None,
+            'weighted_earned': 2.0,
+            'weighted_possible': 4.0,
+            'user_id': 'UserID',
+            'course_id': 'CourseID',
+            'usage_id': 'i4x://org/course/usage/123456',
+            'only_if_higher': False,
+            'score_deleted': False,
```

Comment:
Since you test for `score_deleted` being `False` below, do you want to test for it as `True` here?